src/Elasticsearch.Net/Connection/Content/RequestDataContent.cs

```diff
@@ -195,6 +197,8 @@ protected override void Dispose(bool disposing)
 			public override void EndWrite(IAsyncResult asyncResult) => _innerStream.EndWrite(asyncResult);
 
 			public override void WriteByte(byte value) => _innerStream.WriteByte(value);
+
+			public override void Close() => _innerStream.Close();
```

Comment:
https://github.com/dotnet/corefx/blob/master/src/Common/src/System/IO/DelegatingStream.cs

Does not implement this, I think its good to have though.